apps/commercetools/src/AppConfig/parameters.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { toInputParameters } from "./parameters";
+
+export const definitions = [
```

Comment:
This definitely should be a separate module

Reply:
This should have been imported from parameters.ts but somehow got duplicated so i removed it completely